tests/js/spec/views/settings/components/dataScrubbing/utils.tsx

```diff
@@ -0,0 +1,8 @@
+import convertRelayPiiConfig from 'app/views/settings/components/dataScrubbing/convertRelayPiiConfig';
+
+// @ts-ignore
+const relayPiiConfig = TestStubs.DataScrubbingRelayPiiConfig();
+const stringRelayPiiConfig = JSON.stringify(relayPiiConfig);
+const convertedRules = convertRelayPiiConfig(stringRelayPiiConfig);
+
+export {convertedRules, stringRelayPiiConfig, relayPiiConfig};
```

Comment:
I would suggest against this pattern and instead move this into the test where it is used. The reason being now you have shared state between tests and if you were to mutate the object unintentionally, it can cause some headaches.